pipeline/api/cloud.py

```diff
@@ -55,6 +55,35 @@
     from pipeline.objects import Function, Graph, Model
 
 FILE_CHUNK_SIZE = 200 * 1024 * 1024  # 200 MiB
+BINARY_MIME_TYPE = "application/octet-stream"
+
+
+def _as_upload_file(object, name: Optional[str] = None):
+    """Represent `object` as an HTTP file upload.
+
+    Returns a structure suitable for passing to the `files` parameter of
+    httpx.request.
+    """
+    if name is None:
+        name = str(uuid.uuid4())
+    return (name, object, BINARY_MIME_TYPE)
+
+
+def _ensure_binary_file(file_or_path: Union[BinaryIO, str, pathlib.Path]) -> BinaryIO:
```

Comment:
I can only see this being used in one place (by `_as_binary_file`) so just checking it is actually needed?

Reply:
Good catch! It isn't needed. The `instance(file_or_path, str)` check always failed because the argument is always a `io.BytesIO`.

💣